apps/authentication-portal/src/main/webapp/login.jsp

```diff
@@ -397,18 +397,31 @@
                                             <script src="https://accounts.google.com/gsi/client" async defer></script>
 
                                             <div id="google_parent" class="google-one-tap-container">
-                                                <div id="g_id_onload"
-                                                    data-client_id="<%=Encode.forHtmlAttribute(GOOGLE_CLIENT_ID)%>"
-                                                    data-login_uri="<%=Encode.forHtmlAttribute(GOOGLE_CALLBACK_URL)%>"
-                                                    data-prompt_parent_id="google_parent"
-                                                    data-state="<%=request.getParameter("sessionDataKey")%>"
-                                                    data-cancel_on_tap_outside="false"
-                                                    data-authenticator="<%=Encode.forHtmlAttribute(idpEntry.getValue())%>"
-                                                    data-idp="<%=Encode.forHtmlAttribute(idpName)%>"
-                                                    data-one_tap_enabled="true"
-                                                    data-moment_callback="onMoment">
-                                                </div>
                                             </div>
+
+                                            <form action="<%=GOOGLE_CALLBACK_URL%>" method="post" id="googleOneTapForm">
```

Comment:
we might need to add `display: none;` to this form as well.

Reply:
Fixed 73f0c7d1d762facf2161caf8e55de61c9379dc20